### PR TITLE
Add "phantom" images for outstanding speaker announcements.

### PR DESCRIPTION
**Pull request by cramforce**
This has the added benefit of fixing the layout with a small number of speakers.

### CODE REVIEW

File: templates/pages/speakers.html.njk [hunk 1]
```diff
@@ -27,13 +27,18 @@
...
-    <h3 class="grid-item-left">
-      More speakers are being announced daily on
-      <a href="https://twitter.com/jsconfeu">@jsconfeu</a>.
-    </h3>
-  </section>
+    {% for phantom in range(0, 44 - speakerPages.length) %}
+      <section class="grid-item-speaker phantom">
+        <a class="speaker" href="{{ page.url }}">
+          <figure class="ma0 speaker-picture">
+            <img src="https://placebear.com/400/400">
```
Comment (usefulthink) [inline templates/pages/speakers.html.njk]: really?

File: templates/pages/speakers.html.njk [hunk 2]
```diff
@@ -27,13 +27,18 @@
...
 
-  <section class="grid">
-    <h3 class="grid-item-left">
-      More speakers are being announced daily on
-      <a href="https://twitter.com/jsconfeu">@jsconfeu</a>.
-    </h3>
-  </section>
+    {% for phantom in range(0, 44 - speakerPages.length) %}
+      <section class="grid-item-speaker phantom">
+        <a class="speaker" href="{{ page.url }}">
```
Comment (usefulthink) [inline templates/pages/speakers.html.njk]: oh, and there probably won't be detail pages.

File: templates/pages/speakers.html.njk (hunk 1 above)
Comment (usefulthink) [reply templates/pages/speakers.html.njk]: we should make the numbers unique so we get different bears :)

File: templates/pages/speakers.html.njk (hunk 2 above)
Comment (cramforce) [reply templates/pages/speakers.html.njk]: Fixed.

File: templates/pages/speakers.html.njk (hunk 1 above)
Comment (cramforce) [reply templates/pages/speakers.html.njk]: Wasted data :)

I think this is fine. But feel free to add something like a `?`

**Pull request merged**